lib/new-admin/graphql/schema.js

```diff
@@ -167,9 +194,28 @@ const typeDefs = gql`
   type UserToken {
     token: String!
     name: String!
-    created: Date!
+    created: DateTime!
     user_agent: String
     ip_address: String
+    last_accessed: DateTime
+  }
+
+  type UserSession {
+    sid: String!
+    sess: JSONObject!
+    expire: DateTime!
+  }
+
+  type User {
+    id: ID!
+    username: String!
+    password: String!
+    role: String!
+    enabled: Boolean!
+    created: DateTime!
+    last_accessed: DateTime!
```

Comment:
Why are all of these fields required? Why are we sending password?

Reply:
The password field was probably something on my part on debugging. This users query is only used on the user management screen, where all those fields are required. Is it overkill on my part to put the fields required?